scripts/make/build_musl.mk

```diff
@@ -3,11 +3,15 @@ rust_lib := target/$(TARGET)/$(MODE)/lib$(rust_lib_name).a
 
 musl_version := 1.2.3
 
+build_suffix := _$(ARCH)_$(MODE)
 muslibc_dir := ulib/ruxmusl
-build_dir := $(muslibc_dir)/build_musl_$(ARCH)
+build_dir := $(muslibc_dir)/build_musl$(build_suffix)
+install_dir_name := install$(build_suffix)
+install_dir := $(muslibc_dir)/$(install_dir_name)
+
 musl_dir := $(muslibc_dir)/musl-$(musl_version)
-inc_dir := $(muslibc_dir)/install/include
-c_lib := $(muslibc_dir)/install/lib/libc.a
+inc_dir := $(install_dir)/include
+c_lib := $(install_dir)/lib/libc.a
```

Comment:
Please apply these changes to #69 too! The `install_dir` should be distinguished by `ARCH` too. Thanks!